index.js

```diff
@@ -1,7 +1,14 @@
-require("./lib/profiler")();
+const profile = require("./lib/profiler");
 let isListeningOnStdin = false;
+const defaultOpts = {
+  profilePath: void 0,
```

Comment:
why `void 0`?

Reply:
just to be as-is: `require("./lib/profiler")();`

Reply:
what do you mean by that?

Reply:
atm you are using `require("./lib/profiler")();`, it means `undefined` was passed to `profilePath`.  non-English-speaker, sorry!~

Reply:
What if `require("./lib/profiler")();` returns an object which allows setting a path later?

Reply:
👍 sgtm!